README.md

```diff
@@ -333,6 +333,47 @@ Component.prototype.isReactComponent = {};
 
 :bulb: To assign the `props` you can simply say: `this.props = props;`
 
-## 11. State
+## 12. State
 
-## 12. Conditional rendering
+As mentioned, the whole point of making this Component class is to be able to create stateful components.
+So finally, let's add some state.
+
+Setting the initial state of your component is really easy. Just assign an object with some properties to the property `state` on your class.
+Just like with props, this is now accessible through `this.state`.
+
+```jsx
+class Greeting extends React.Component {
+    state = { name: "world" };
```

Comment:
Verdt å merke seg at man må bruke Babel for å sette klasse-variabler slik som dette, da det ikke er en del av es2015

Reply:
Aha. Jeg googla kjapt i går, og fant noe om at noe var i es2015. Jeg trodde det var dette, men når jeg tenker meg om så var det nok ikke det. Da setter jeg dette i konstruktøren tenker jeg.